services/userService.js

```diff
@@ -36,6 +37,41 @@ const signup = async (email, password, name, birth, phoneNumber, address) => {
   }
 };
 
+const login = async (email, password) => {
+  try {
+    const hashedPassword = await userDao.getUserPasswordByEmail(email);
+    const checkHash = await bycrpt.compare(password, hashedPassword);
+
+    if (!checkHash) {
+      const err = new Error("WRONG_PASSWORD");
+      err.statusCode = 400;
+      throw err;
+    }
+    const currentTime = currentUtcKoreaTime();
+    const ExpireTime = currentTime + 60 * 60 * 24;
+    const userId = await userDao.checkRegisteruserId(email);
+    const payload = {
+      iss: "garbageCollectoOwner",
+      sub: "garbageWorld",
+      iat: currentTime,
+      exp: ExpireTime,
+      userId: userId,
+    };
+    const token = jwt.sign(payload, process.env.SECRET_KEY);
+    return token;
```

Comment:
```suggestion
    const token = await jwt.sign(payload, process.env.SECRET_KEY);
    return token;
```

위와 같이 await가 걸려있어야 보다 안전하게 오작동 없이, 동기적으로 동작할 것 같습니다. 
혹은 아래 예시와 같이 곧바로 리턴해주면 더 좋을 것 같아요!

```javascript
    return jwt.sign(payload, process.env.SECRET_KEY);
```